router/batchrouter/batchrouter.go

```diff
@@ -1797,7 +1797,7 @@ func (worker *workerT) workerProcess() {
 				switch {
 				case misc.Contains(objectStorageDestinations, brt.destType):
 					destUploadStat := stats.Default.NewStat(fmt.Sprintf(`batch_router.%s_dest_upload_time`, brt.destType), stats.TimerType)
-					destUploadStat.Start()
+					defer destUploadStat.RecordDuration()()
```

Comment:
it would me more appropriate to not use a defer here and clearly define the timer's start and end
```
start := time.Now()
...
stat.Since(start)
```